spec/models/exercise_spec.rb

```diff
@@ -1,10 +1,42 @@
 require 'rails_helper'
 
 RSpec.describe Exercise, type: :model do
+  let(:current_user) { FactoryBot.create(:user) }
+  let(:exercise) { FactoryBot.create(:exercise, user: current_user) }
+
+  describe 'Basic setup methods' do
+    it 'has a valid factory' do
+      expect(exercise).to be_valid
+    end
+
+    it 'persists a valid model object when given strings for title, general description, user id and a global boolean' do
+      exercise = Exercise.new(
+        title: 'Senecas Dance with Dragons',
+        general_description: 'An amazing exercise to really get your pants off.',
+        user: current_user
+      )
+    end
+  end
+
+  describe 'Classic ActiveRecord Instance Methods' do
+    it 'returns a title' do
+      expect(exercise.title).to eq('Senecas Courage Walk')
+    end
+
+    it 'returns a general description' do
+      expect(exercise.general_description).to eq('Take a stroll with Seneca and beef up your courage.')
+    end
+
+    it 'returns a global boolean' do
+      expect(exercise.global?).to eq(false)
+    end
```

Comment:
I don't think it's very good style to test values "magically" appearing from the factory. This is brittle should you need to change them for any reason. If you're testing for values set them explicity during construction of the object.

Reply:
Yes great point. I will keep this in mind in the future.

I updated these 3 tests to manually assign the relevant value inside the test directly before the test.